pkg/foghorn/controller.go

```diff
@@ -422,9 +429,14 @@ func (c *Controller) getReportURLBase() string {
 	return os.Getenv("LIGHTHOUSE_REPORT_URL_BASE")
 }
 
+// getReportURLTeam gets the team to construct the report url
+func (c *Controller) getReportURLTeam() string {
+	return os.Getenv("LIGHTHOUSE_REPORT_URL_TEAM")
```

Comment:
What happens if this env variable is not set?

Should we guard against that or will it always be present?